### PR TITLE
add Java TF Example extractor

### PR DESCRIPTION
**Pull request by nevillelyh**

### CODE REVIEW

File: java/src/main/java/com/spotify/featran/java/JFeatureExtractor.java [hunk 1]
```diff
@@ -79,4 +80,11 @@ public String featureSettings() {
...
 
-}
+  /**
+   * Java wrapper for {@link FeatureExtractor#featureValues(FeatureBuilder, ClassTag)}.
+   */
+  public List<Example> featureValuesExample() {
+    return JavaOps.featureValuesExample(self);
+  }
+
+}
```
Comment (ravwojdyla) [inline java/src/main/java/com/spotify/featran/java/JFeatureExtractor.java]: `+` newline?

Comment (nevillelyh) [reply java/src/main/java/com/spotify/featran/java/JFeatureExtractor.java]: DAMN IntelliJ

**Comment by nevillelyh** [comment]
This will pull in TF proto dependency though (right now 1.4.0) but it should be stable enough and not cause conflicts?

**Comment by nevillelyh** [comment]
OTOH none the TF artifacts, e.g. `org.tensorflow:{proto,tensorflow,libtensorflow}` has any transitive dependency. Maybe a plus side of them using bazel?

**Comment by ravwojdyla** [comment]
@nevillelyh is it only for the `tf.Example`? we could also make it `provided`?

**Comment by nevillelyh** [comment]
I think that will break those not using `tf.Example` output and without an explicit dependency, since the classloader will complain when loading.
Keep it in and let users override version (easy)?

**Comment by ravwojdyla** [comment]
ah, yes - i guess it will, sure.

**Pull request merged**